<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdjustPlotVisualsDialog</class>
 <widget class="QDialog" name="AdjustPlotVisualsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adjust plot appearance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="qgb_series">
     <property name="title">
      <string>Series</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="ql_series">
          <property name="text">
           <string>Serie</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="qcbox_series"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="ql_lineColor">
          <property name="text">
           <string>Line color</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="ql_lineColorClr">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 0.2em</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qpb_pickLineColor">
            <property name="text">
             <string>Pick</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="ql_lineThickness">
          <property name="text">
           <string>Line thickness</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="qspbox_lineThickness">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="ql_pointColor">
          <property name="text">
           <string>Point color</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="ql_pointSize">
          <property name="text">
           <string>Point size</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="ql_pointColorClr">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 0.2em</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qpb_pickPointColor">
            <property name="text">
             <string>Pick</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="qspbox_pointSize">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ql_pointStyle">
          <property name="text">
           <string>Point style</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="qcbox_pointStyles"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="ql_pointLineThickness">
          <property name="text">
           <string>Point line thinckness</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="ql_pointFillColor">
          <property name="text">
           <string>Point fill color</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDoubleSpinBox" name="qspbox_pointLineThickness">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="ql_pointFillColorClr">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qpb_pickPointFillColor">
            <property name="text">
             <string>Pick</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qgb_axes">
     <property name="title">
      <string>Axes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="ql_axisFontSize">
          <property name="text">
           <string>Font size</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="qspbox_axisFontSize">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="ql_axis">
          <property name="text">
           <string>Axis</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="qcbox_axis"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="ql_axisBold">
          <property name="text">
           <string>Bold</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="qcb_axisBold">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="qpb_setForAllAxes">
        <property name="text">
         <string>Set for all axes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="qpb_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>qcbox_series</tabstop>
  <tabstop>qcbox_pointStyles</tabstop>
  <tabstop>qpb_pickLineColor</tabstop>
  <tabstop>qspbox_lineThickness</tabstop>
  <tabstop>qpb_pickPointColor</tabstop>
  <tabstop>qspbox_pointSize</tabstop>
  <tabstop>qcbox_axis</tabstop>
  <tabstop>qspbox_axisFontSize</tabstop>
  <tabstop>qcb_axisBold</tabstop>
  <tabstop>qpb_setForAllAxes</tabstop>
  <tabstop>qpb_ok</tabstop>
  <tabstop>qpb_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
